iothub/device/src/Transport/Mqtt/ClientWebSocketChannel.cs

```diff
@@ -262,9 +260,22 @@ private async Task HandleCloseAsync()
 
         private void Abort()
         {
-            _webSocket.Abort();
-            _webSocket.Dispose();
-            _writeCancellationTokenSource.Dispose();
+            _webSocket?.Abort();
+            _webSocket?.Dispose();
+            _webSocket = null;
+
+            _writeCancellationTokenSource?.Dispose();
+            _writeCancellationTokenSource = null;
+        }
+
+        /// </inheritdoc>
+        public void Dispose()
```

Comment:
we should dispose this clientwebsocket channel within the mqtt transport handler dispose call